flang/lib/Optimizer/Transforms/AffinePromotion.cpp

```diff
@@ -504,6 +533,9 @@ class AffineDialectPromotion
 
     auto *context = &getContext();
     auto function = getFunction();
+    markAllAnalysesPreserved();
+    Liveness l = Liveness(function);
+    // l.dump();
```

Comment:
LLVM_DEBUG() ?

Reply:
I just added this here when I was playing around with liveness. I will remove it.